digitaltwins/samples/model_repository/get_model.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+const ModelRepositoryCredentials = require('azure-iot-digitaltwins-model-repository').ModelRepositoryCredentials;
+const ModelRepositoryServiceClient = require('azure-iot-digitaltwins-model-repository').ModelRepositoryServiceClient;
+
+function replacer(key, value) {
+  if (key == '_response') {
+    return undefined;
+  } else {
+    return value;
+  }
+}
+
+async function main() {
+  const modelRepositoryCredentials = new ModelRepositoryCredentials(process.env.AZURE_IOT_MODEL_REPOSITORY_CONNECTION_STRING);
```

Comment:
>AZURE_IOT_MODEL_REPOSITORY_CONNECTION_STRING [](start = 80, length = 44)

I think the "AZURE_IOT_" is unnecessary here

Reply:
That is by design. It is a little bit of branding :)

---
In reply to: [337239505](https://github.com/Azure/azure-iot-sdk-node/pull/646#discussion_r337239505) [](ancestors = 337239505)